src/pybind/mgr/mgr_module.py

```diff
@@ -105,13 +105,16 @@ def wait(self):
         return self.r, self.outb, self.outs
 
 
-class HandleCommandResult(namedtuple('HandleCommandResult', ['retval', 'stdout', 'stderr'])):
+class HandleCommandResult(namedtuple('HandleCommandResult', ['retval', 'odata', 'rs'])):
```

Comment:
i'd lean toward keeping the stdout/stderr naming here so it's clear what the fields are for.  'odata' and 'rs' mirror common variable naming in the monitor C++ code but are mostly meaningless...

Reply:
I had my problems especially with `stderr`, as it is also used for successful status messages. E.g. something was created successfully. If we want to change this, I'd more lean towards `data` and `message` here.

Reply:
I think we should work to remove commands that return "i did foo" in the stderr stream; that should be implied by the successful exit code and lack of an error message

Reply:
Would you prefer to use `data=` here?
https://github.com/ceph/ceph/blob/fb48a775b59f44e4ec91e9d1515c072ef36bff1d/src/pybind/mgr/orchestrator_cli/module.py#L196

Reply:
in that case IMO it should either be odata= (or stdout= :), or there should be an error/exit code.  My thinking is

- only write to stderr if there is an error, or in extraordinary circumstances
- avoid having 'ceph foo bar' commands say "did foo bar" on success unless there is critical information to include there.. and if there is something that you would programmatically consume, put it on stdout, not stderr.

Reply:
In this case, `stdout` and `stderr` does make sense.

Reply:
i am also for `stdout` and `stderr`. 

[OT] and we might want to update the C++ counterpart. odata and rs deserve better names =)

Reply:
done.